changelog.d/13729.misc

```diff
@@ -0,0 +1 @@
+Strip number suffix from instance name to consolidate services that traces are spread over.
```

Comment:
@clokep Reviewer wheel chose you 🙇 or leave it in the queue if it will be picked up. Just didn't want to leave it sit with no action since there was no review transition for all of the discussion.

Reply:
Due to the amazing thing that is timezones, @erikjohnston got to it before me. 🎉